src/shared/fargate.ts

```diff
@@ -2,7 +2,7 @@ import { cpuStyle, memoryStyle, weekBeforeStyle } from "./styles";
 import { Component } from "../index";
 
 // fargate ecs stacks, not plain ECS
-export default function fargate(ecs_task_family: string): Component {
+export default function fargate(tag_key: string, tag_value: string): Component {
```

Comment:
This change will break all the existing monitors without updating them to use these new params – is this PR more of  an RFC before updating further?

Seems like this change will make the default interface not as intuitive when wiring up a new service. My preference would be leaning towards passing through `service` rather than tags or `ecsFamily` to ensure we get decent monitors hooking into our [Datadog Unified Service Tagging](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/).

Also, small thing, we should use camel casing for TypeScript!

Reply:
> My preference would be leaning towards passing through service rather than tags or ecsFamily to ensure we get decent monitors hooking into our Datadog Unified Service Tagging.

That's a good point. It might be better to set up apps to support the Unified Service Tagging rather than making this change.

I'll close this PR.